src/go.mod

```diff
@@ -0,0 +1,28 @@
+module github.com/containers/toolbox
+
+go 1.13
+
+require (
+	github.com/HarryMichal/go-version v1.0.0
+	github.com/StackExchange/wmi v0.0.0-20190523213315-cbe66965904d // indirect
+	github.com/briandowns/spinner v1.9.0
+	github.com/cpuguy83/go-md2man/v2 v2.0.0
+	github.com/fatih/color v1.9.0 // indirect
+	github.com/go-ole/go-ole v1.2.4 // indirect
+	github.com/godbus/dbus/v5 v5.0.3
+	github.com/konsorten/go-windows-terminal-sequences v1.0.2 // indirect
+	github.com/mattn/go-colorable v0.1.6 // indirect
+	github.com/pelletier/go-toml v1.6.0 // indirect
+	github.com/shirou/gopsutil v2.20.2+incompatible
+	github.com/sirupsen/logrus v1.4.2
+	github.com/spf13/afero v1.2.2 // indirect
+	github.com/spf13/cast v1.3.1 // indirect
+	github.com/spf13/cobra v0.0.6
+	github.com/spf13/jwalterweatherman v1.1.0 // indirect
+	github.com/spf13/pflag v1.0.5 // indirect
+	github.com/spf13/viper v1.6.2
```

Comment:
I think that we should avoid the dependency on `github.com/spf13/viper`, because that reduces the size of the `toolbox` binary from 14 MB to 7.6 MB - almost by half.

In many cases, we can replace it with `os.Getenv` and `os.Setenv`.Then we can have a `podman.SetLogLevel` function that updates a global variable to implement the Podman logging.

Ultimately, we don't need a fully featured configuration management framework, because we almost don't have any settings at the moment. It can, of course, change in the future.